client/js/store/map/Layers.js

```diff
@@ -109,11 +110,11 @@ const LayersStore = {
       }
 
       // Get index of data in relationships based on above switch
-      indexRelationships = relationships.findIndex(elem => elem.id === element.id)
+      indexRelationships = relationships.FindIndex(elem => elem.id === element.id)
```

Comment:
too eager :D
```suggestion
      indexRelationships = relationships.findIndex(elem => elem.id === element.id)
```

Reply:
Thanks for spotting 852ab24aea8726fda86706a749b7c6bd90ed2269